### ISSUE TITLE
Inappropriate ioctl for device

### ISSUE THREAD

**Issue by bray12345giff** [alubbock/rpi-vidlooper#5]
I have the same equipment and setup the Raspberry Pi per your instructions, but I am getting this error:

using "DejaVu Sans Mono-16", pixelsize=16.67 file=/usr/share/fonts/truetype/dejavu/DejaVuSansMono.ttf
ioctl VT_GETSTATE: Inappropriate ioctl for device (not a linux console?)

When I use this command:

vidlooper --splash /home/pi/splash.png --no-loop --countdown=5 --video-dir /home/pi/videos

Any ideas as to what would be causing this?

**Comment by alubbock** [comment]
Hi, are you running this command over `ssh`? The splash screen uses a command called `fbi`, which will only work if it has access to the local terminal (i.e. is not run inside `ssh`, `screen`, `tmux` or similar programs).

**Comment by bray12345giff** [comment]
Yes thank you, that was the problem. Everything is working as expected, except for running vidlooper at login. I have added my vidlooper command to the end of bash.rc, but this is printed: -bash: vidlooper: command not found. I can run the command manually and it works fine though. I am wondering if it has something to do with the PATH? I did have to update the path once I installed vidlooper as it wasn't in the PATH, but I'm not sure how to fix this
